sequence_processing_pipeline/tests/test_Job.py

```diff
@@ -288,6 +288,22 @@ def test_wait_on_job_ids(self):
         # query_slurm(), they should be equal.
         self.assertDictEqual(obs, results)
 
+    def test_mark_completed_commands(self):
+        package_root = abspath('./sequence_processing_pipeline')
+        self.path = partial(join, package_root, 'tests', 'data')
+
+        job = Job(self.path('211021_A00000_0000_SAMPLE'),
+                  self.path('my_output_dir'), '200nnn_xnnnnn_nnnn_xxxxxxxxxx',
+                  ['ls'], 2, None)
+
+        fp = join(job.output_path, 'job_completed')
+
+        self.assertFalse(exists(fp))
+
+        job.mark_job_completed()
+
+        self.assertTrue(exists(fp))
```

Comment:
Mark job completed creates a file? File based interprocess communication is not safe...

Reply:
The concern here is files on network mounts are not assured to be visible to all systems when expected. Is there a large (minutes) time delay for when an attempt to read one of these files occurred from another system?

Reply:
A solid point, but you are correct a failed job is always reviewed and restarted manually so the time delay should be sufficient; the working directories are mounted on Lustre (DDN) if I recall correctly.